systemd/manifest.json

```diff
@@ -22,7 +22,7 @@
   "assets": {
     "monitors": {},
     "dashboards": {
-      "Systemd Overview": "assets/dashboards/systemd_overview.json"
+      "Systemd Overview": "assets/dashboards/overview.json"
```

Comment:
can it be called just overview?

Reply:
I think it can be named anything, but of course we should be consistent. Here are the names we currently use:

```
; find . -name '*.json' | grep 'dashboards\/.*\.json'
./presto/assets/dashboards/overview.json
./ibm_db2/assets/dashboards/overview.json
./ambari/assets/dashboards/base_dashboard.json
./harbor/assets/dashboards/overview.json
./nginx/assets/dashboards/plus_overview.json
./hive/assets/dashboards/overview.json
./cockroachdb/assets/dashboards/overview.json
```

Reply:
However the dashboard "key" `"Systemd Overview"` must be unique.